CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # gulp-static-i18n changelog
 
+## 0.0.2
+
+* Allow sourcing of PO files from Multiple directories
```

Comment:
oh right, forgot about that. in that case this should be `1.0.0`, shouldn't it? since it's a guaranteed breaking change?